website/pages/en/global.json

```diff
@@ -10,5 +10,20 @@
   "notFoundTitle": "Oops! This page was lost in space...",
   "notFoundSubtitle": "Check if you’re using the right address or explore our website by clicking on the link below.",
   "goHome": "Go Home",
-  "video": "Video"
+  "video": "Video",
+  "sidebar": {
```

Comment:
Should this be named `nav` instead of `sidebar`?

Reply:
Hmm, I guess `sidebar` is consistent with Nextra's naming of `sidebarTitle`. Just seems a little too concerned about the UI. :P